src/stores/rootStore.ts

```diff
@@ -217,7 +217,8 @@ export const useRootStore = defineStore('root', {
         if (!dissolutionDate) throw new Error('Invalid dissolution date')
         date = DateUtilities.dateToPacificDate(dissolutionDate, true)
       } else {
-        name = EnumUtilities.filingTypeToName(filingType)
+        name = filingType === FilingTypes.CONTINUATION_OUT
+          ? 'Continued Out' : EnumUtilities.filingTypeToName(filingType)
```

Comment:
A ternary like this is appropriate here 👍 but parentheses around the boolean would make this easier to read.

Another thing that works well is this format:
```
name = (filingType === FilingTypes.CONTINUATION_OUT)
  ? 'Continued Out'
  : EnumUtilities.filingTypeToName(filingType)
```

Or something like that. Up to you.